<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DDS Converter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="conversionModeComboBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>DXT1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DXT3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DXT5</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="textureOrientationComboBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>DirectX</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>OpenGL</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="conversionModeLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Conversion mode:</string>
    </property>
   </widget>
   <widget class="QLabel" name="textureOrientationLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Texture orientation:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="processedAssetsTable">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>511</width>
      <height>331</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>120</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>File</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Compression type</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="convertImageButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="convertFolderButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert folder</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="processingProgressBar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>511</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QLabel" name="currentAssetLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>511</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>575</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
